src/Opulence/Views/Compilers/Fortune/Lexers/Lexer.php

```diff
@@ -604,4 +604,24 @@ function_exists($token[1])
 
         return trim($joinedTokens);
     }
+
+    /**
+     * Polyfill for PHP each, which is now deprecated
+     *
+     * @param array $arr
+     *
+     * @return array|bool
```

Comment:
Should this be `@return array|null`?  Can it ever return a bool?

Reply:
Fixed.